src/classes/metadata/pdf.vala

```diff
@@ -229,7 +229,7 @@ namespace pdfpc.Metadata {
                     GLib.FileUtils.set_contents(this.pdfpc_fname, contents);
```

Comment:
I can't comment on line 227, but currently we silently fail if `GLib.FileUtils.test(this.pdfpc_fname, (GLib.FileTest.IS_REGULAR))` return false. Maybe we can have something like

```
// nothing to write here
if (contents == "") {
    return;
}
if (!GLib.FileUtils.test(this.pdfpc_fname, (GLib.FileTest.IS_REGULAR))) {
   throw FileError(); // I have no idea about the syntax :)
}
GLib.FileUtils.set_contents(this.pdfpc_fname, contents);
```
The catch clause is good.